readme.md

```diff
@@ -96,6 +98,12 @@ The swagger files are compiled, and then run, which pings the mock server (to ve
 
 This document contains some additional [technical details](https://github.com/Azure/autorest.csharp/blob/feature/v3/test/README.md).
 
+**cadl-ranch**
+
+[cadl-ranch](https://github.com/Azure/cadl-ranch) is similar to autorest.testserver, which provides a platform for automated testing too. The difference is the testing target of cadl-ranch is typespec generated SDK.
+
+To test local change in cadl-ranch, run `.\eng\ApplyCadlRanch.ps1 {cadl-ranch path}` or just run `.\eng\ApplyCadlRanch.ps1` if cadl-ranch folder is at the same folder of autorest.csharp.
```

Comment:
From the above script, seems ApplyCadlRanch.ps1 only handles the file copy work.  Do you have the E2E steps that we test local changes for CADL-ranch?

Reply:
Yes, copy is far enough. Think about how we test local .net code: we build our local change into the assembly, find out the place where our product loads the assembly, and copy the assembly with our change into that place, period.

So it is the similar process for ts code. We build code into some executable files, copy them to where node loads these files. Then just do whatever you want.